integrated_channels/sap_success_factors/migrations/0001_squashed_0022_auto_20200206_1046.py

```diff
@@ -0,0 +1,96 @@
+# Generated by Django 2.2.12 on 2020-04-21 18:57
+
+from django.conf import settings
+from django.db import migrations, models
+import django.utils.timezone
+
+import model_utils.fields
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('sap_success_factors', '0001_initial'), ('sap_success_factors', '0002_auto_20170224_1545'), ('sap_success_factors', '0003_auto_20170317_1402'), ('sap_success_factors', '0004_catalogtransmissionaudit_audit_summary'), ('sap_success_factors', '0005_historicalsapsuccessfactorsenterprisecustomerconfiguration_history_change_reason'), ('sap_success_factors', '0006_sapsuccessfactors_use_enterprise_enrollment_page_waffle_flag'), ('sap_success_factors', '0007_remove_historicalsapsuccessfactorsenterprisecustomerconfiguration_history_change_reason'), ('sap_success_factors', '0008_historicalsapsuccessfactorsenterprisecustomerconfiguration_history_change_reason'), ('sap_success_factors', '0009_sapsuccessfactors_remove_enterprise_enrollment_page_waffle_flag'), ('sap_success_factors', '0010_move_audit_tables_to_base_integrated_channel'), ('sap_success_factors', '0011_auto_20180104_0103'), ('sap_success_factors', '0012_auto_20180109_0712'), ('sap_success_factors', '0013_auto_20180306_1251'), ('sap_success_factors', '0014_drop_historical_table'), ('sap_success_factors', '0015_auto_20180510_1259'), ('sap_success_factors', '0016_sapsuccessfactorsenterprisecustomerconfiguration_additional_locales'), ('sap_success_factors', '0017_sapsuccessfactorsglobalconfiguration_search_student_api_path'), ('sap_success_factors', '0018_sapsuccessfactorsenterprisecustomerconfiguration_show_course_price'), ('sap_success_factors', '0019_auto_20190925_0730'), ('sap_success_factors', '0020_sapsuccessfactorsenterprisecustomerconfiguration_catalogs_to_transmit'), ('sap_success_factors', '0021_sapsuccessfactorsenterprisecustomerconfiguration_show_total_hours'), ('sap_success_factors', '0022_auto_20200206_1046')]
```

Comment:
@dianakhuang: In `0010_move_audit_tables_to_base_integrated_channel`, a Model was moved to integrated_channel, and then ultimately removed I think, so I am squashing both these apps in one PR to remove all this.

Reply:
FYI: In integrated_channel, the model `CatalogTransmissionAudit` was [created in 0003](https://github.com/edx/edx-enterprise/blob/c71727bd465e32ad9649fbb43f78b5a5cdbe0d83/integrated_channels/integrated_channel/migrations/0003_catalogtransmissionaudit_learnerdatatransmissionaudit.py#L14-L21) (moved here from sap_success_factors 0010) and [deleted in 0006](https://github.com/edx/edx-enterprise/blob/50e42c768e0de3dbe1451f512fd24b0588f4973c/integrated_channels/integrated_channel/migrations/0006_delete_catalogtransmissionaudit.py#L15).